book/week_02/01-Language-Fundamentals.ipynb

```diff
@@ -1042,7 +1042,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is immutability which makes tuples useful, but for beginners this is not really obvious at the first sight. We will get back to tuples later in the lecture."
+    "It is immutable which makes tuples useful, but for beginners this is not really obvious at the first sight. We will get back to tuples later in the lecture."
```

Comment:
I disagree on this one, but will revert the change locally

Reply:
Oh well, reads way better how it is now!